components/server/src/user/user-deletion-service.ts

```diff
@@ -113,6 +113,12 @@ export class UserDeletionService {
             runningWorkspaces.map(async (info) => {
                 const wsi = info.latestInstance;
 
+                const reason = "deleting user";
+                log.info(
+                    { workspaceId: info.workspace.id, instanceId: wsi.id },
+                    `Stopping workspace instance (reason: ${reason})`,
```

Comment:
:bulb: Could also be a payload, so it's easier to write log queries for it:

`..., "Stopping workspace instance", { reason });`